### PR TITLE
Package semver2.1.2.0

### PR DESCRIPTION
**Pull request by knuton**
### `semver2.1.2.0`
Semantic version handling for OCaml

---
* Homepage: https://github.com/dividat/ocaml-semver
* Bug tracker: https://github.com/dividat/ocaml-semver/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/semver2/semver2.1.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {>= "1.2.0"}
+  "angstrom" {>= "0.14.0"}
+  "ounit" {with-test & >= "1.0.2"}
+  "yojson" {with-test & >= "1.4.1"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/semver2/semver2.1.2.0/opam:17]:
```suggestion
  ["dune" "subst"] {dev}
```

File: packages/semver2/semver2.1.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
```
Comment (mseri) [inline packages/semver2/semver2.1.2.0/opam:20]: The url stanza pointing at the sources is missing

Comment (mseri) [reply packages/semver2/semver2.1.2.0/opam:20]: Try `opam-publish https://github.com/dividat/ocaml-semver/archive/refs/tags/1.2.0.tar.gz` it should take care of that.

File: packages/semver2/semver2.1.2.0/opam (hunk 1 above)
Comment (mseri) [reply packages/semver2/semver2.1.2.0/opam]: Please make this change also upstream (no need to re-tag or re-release)

**Comment by mseri** [comment]
Thanks!

**Comment by mseri** [comment]
Feel free to announce the release on the discuss.ocaml.org forum, we have a community category and an announcement tag for this purpose

File: packages/semver2/semver2.1.2.0/opam (hunk 1 above)
Comment (knuton) [reply packages/semver2/semver2.1.2.0/opam]: Is there some context for this?

`dune`'s docs still prescribe `pinned`: https://github.com/ocaml/dune/blob/2.9/doc/opam.rst#invocation-from-opam

Comment (knuton) [reply packages/semver2/semver2.1.2.0/opam]: Nevermind, just saw your PR in the dune project. 👍

**Comment by knuton** [comment]
Thank you! I backported the fixes from this PR to our repo.

**Pull request merged**